Fixed mixture of Kong schema and Draft 4 (#4087)
Kong/docs.konghq.com: app/_hub/kong-inc/request-validator/_index.md

    }
    ```

> 4. In these step examples, validation makes sure that `status_code` is a number and that the body contains a parameter called `name`

Reviewer: ```suggestion
4. In these step examples, validation ensures that `status_code` is a number and the body contains a parameter called `name`.
```

   A proxy request with a non-numerical status code is blocked:

    ```

>     curl --request POST \

Reviewer: ```suggestion
    curl -i -X POST \
```
Using the shorthand for a request + headers for consistency with the rest of the docs site.

    A proxy request with a numeric status code is allowed:

    ```

>     curl --request POST \

Reviewer: ```suggestion
    curl -i -X POST \
```
